salt/common/tools/sbin/so-status

```diff
@@ -101,9 +103,13 @@ create_expected_container_list() {
 }
 
 populate_container_lists() {
+    # TODO: check exit code directly, not with $?
```

Comment:
This was for legibility but could just as easily be written to check the exit code directly.